### DISCUSSION TITLE
View refresh processing commands in All Queries trace?

### DISCUSSION THREAD

**Discussion by match-gabeflores** [DaxStudio/DaxStudio#774] [Ideas]
It seems like Refresh commands (processing a database/table, etc) is not viewable in DAX Studio. I currently have to use SQL Server Profiler. 

Is there a specific reason for that? It would be very convenient to have it all in one place. Thanks for a great product!

![image](https://user-images.githubusercontent.com/286903/146030253-1d72d02f-123b-4369-8254-8255e6e2d8c6.png)

**Comment by dgosbell** [comment]
Sorry for the delay in replying. The main reason these events are not visible yet in DAX Studio is just a lack of spare time to implement this :) I have a list of ideas for new features and the ability to trace refresh events is definitely something that is on my roadmap. I've actually started prototyping something similar to Phil Seamark's PBI report that analyzes the performance of data refreshes. I just have a few other big features that are in flight at the moment that need to be finished off first.

**Reply by match-gabeflores** [reply]
Thanks for the update. Looking forward to it whenever it does arrive!
